src/internal/connector/onedrive/collections.go

```diff
@@ -349,7 +345,18 @@ func (c *Collections) Get(
 			numDeltas)
 
 		if !delta.Reset {
-			maps.Copy(excludedItems, excluded)
+			p, err := GetCanonicalPath(
+				fmt.Sprintf(rootDrivePattern, driveID),
+				c.tenant,
+				c.resourceOwner,
+				c.source)
+			if err != nil {
+				return nil, nil,
+					clues.Wrap(err, "making exclude prefix for drive").WithClues(ctx).With("drive_id", driveID)
+			}
+
+			excludedItems[p.String()] = excluded
```

Comment:
I'm using `driveID` here in my PR.  Assuming all the ids in excluded are unique, could the path creation here get shortened to just that?

Reply:
in theory yes, but that won't play well with kopia because it would turn into a `strings.Contains` calls and that makes me very nervous 😅